API/Controllers/UserController.cs

```diff
@@ -216,8 +221,18 @@ public async Task<IActionResult> DeleteAccount(int userId)
                 return NotFound(problem);
             }
 
+            List<Project> projects = await projectService.GetAllWithUsersAsync(new ProjectFilterParams() { });
+            projects.ForEach(delegate (Project project) {
```

Comment:
change the foreach delegate to a for loop.

_Modifying the underlying collection in the body of the Action<T> delegate is not supported and causes undefined behavior._
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.foreach?view=netcore-3.1

Reply:
Why are you not fetching the projects for the specific user id you are looking for? This would seem like a more straightforward way and you have less logic in you controller.

Reply:
Can't seem to find a method for specifically retrieving all projects bound to a user?

Reply:
I that scenario add a new method to the interface of the service and the repo and implement it